app/src/main/java/com/javainiai/chefskiss/ui/recipescreen/RecipeDetailsViewModel.kt

```diff
@@ -59,6 +59,12 @@ class RecipeDetailsViewModel(
             recipesRepository.insertShopRecipe(ShopRecipe(uiState.value.recipe.id))
         }
     }
+    fun updateFavorite() {
+        val updatedRecipe = uiState.value.recipe.copy(favorite = !uiState.value.recipe.favorite)
+        viewModelScope.launch {
+            recipesRepository.updateFavoriteRecipe(updatedRecipe)
```

Comment:
I am not sure why did we need a separate method added in the database, when an update method already exists? 

```kotlin
recipesRepository.updateRecipe(updatedRecipe)
```